docs/release/release_0_3_2.md

```diff
@@ -0,0 +1,67 @@
+# napari 0.3.2
+
+We're happy to announce the release of napari 0.3.2!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+
+## New Features
+- General multithreading API and @thread_worker decorator (#1210)
+- Rich jupyter display for napari screenshots (#1269)
+
+## Improvements
+- make Qt component non private module (#1122)
+- add option to disable overwrite of labels during painting (#1264)
+- Refactor: move dask utils from misc.py to new dask_utils.py (#1265)
```

Comment:
Generally I would reserve this section for user-facing improvements. Should we move this to an "other improvements" section?

Reply:
Let's save for next time, i did fix the capitolizations